app/helpers/api/integrations_helper.rb

```diff
@@ -101,7 +101,8 @@ def promote_to_staging_button_options(proxy)
   end
 
   def promote_to_production_button_options(proxy)
-    return disabled_promote_button_options if proxy.environments_have_same_config?
+    # FIXME: Hot fix to https://issues.jboss.org/browse/THREESCALE-3760
```

Comment:
issue is about promote to staging button and this changes behavior promote to production button